tests/fixtures/TailwindTestKernel.php

```diff
@@ -39,6 +39,10 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
             'secret' => 'foo',
             'test' => true,
             'http_method_override' => true,
+            'handle_all_throwables' => true,
+            'php_errors' => [
+                'log' => true,
+            ],
```

Comment:
This fix the latest deprecations on highest Symfony dependencies, which made some CI jobs fail